### ISSUE TITLE
Assign Default account to an instance

### ISSUE THREAD

**Issue by RaXorX** [MultiMC/MultiMC5#3928]
### Role

Multiple Account instancing

### Suggestion

Assign default account to an instance.

### Benefit

Easier to handle account management per instance, otherwise you need to change the account in menu first before starting an instance.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by jotoho** [comment]
Duplicate of #3411 and quite a few others

**Comment by RaXorX** [comment]
Damn, yeah bringing in money in the regards of development wasn't such a good idea. I hope in the future maybe it's picked up on once again, by that I mean not rejected.
But that's fair enough, just setting no default works just as fine at the moment.
